contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics.py

```diff
@@ -77,6 +77,39 @@ def get_process_private_bytes_metric():
     return gauge
 
 
+def get_process_cpu_usage():
+    try:
+        # In the case of a process running on multiple threads on different CPU
+        # cores, the returned value of cpu_percent() can be > 100.0. We
+        # normalize the cpu process using the number of logical CPUs
+        cpu_count = psutil.cpu_count(logical=True)
```

Comment:
I'm not sure if we want to get cpu_count each time. Do we optimize for plug-and-play scenario?

Reply:
Good point. I can store the cpu_count information.

Reply:
@reyang  I will be refactoring the standard metrics into their own files in the next PR, so I can do this change then. I think this one is good to merge!